scripts/blocklist-skylink.sh

```diff
@@ -3,14 +3,49 @@
 set -e # exit on first error
 
 if [ -z "$1" ]; then
-    echo "Please provide a skylink to blocklist" && exit 1
+    echo "Please provide either a skylink or file with skylinks separated by blank lines" && exit 1
```

Comment:
```suggestion:
   echo "Please provide either a skylink or file with skylinks separated by new lines" && exit 1
```

Reply:
```suggestion
    echo "Please provide either a skylink or file with skylinks separated by new lines" && exit 1
```

Reply:
Oh, I see. When typing that up I shouldn't add `:` after `suggestion`. But still, it would be easier to remember that there's a button. Thanks. :)